LAMP/SearchContacts.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+	$inData = getRequestInfo();
+
+	$searchResults = "";
+	$searchCount = 0;
+
+	$conn = new mysqli("localhost", "TheBeast", "WeLoveCOP4331", "COP4331");
+	if ($conn->connect_error)
+	{
+		returnWithError( $conn->connect_error );
+	}
+	else
+	{
+		$stmt = $conn->prepare("select FirstName from Contacts where FirstName like ? and UserID=?");

```

Comment:
```suggestion
		$stmt = $conn->prepare("SELECT * FROM Contacts WHERE FirstName LIKE ? AND UserID=?");

```

Reply:
* gives us all the information from the table regarding any relation to that person which is probably what we need for the details panel. I also capitalized the key selector words just for good programming practices.